### PR TITLE
fix: Remove terms 

### LINKED ISSUE #147: Fix various style issues and typos throughout the guide
- [x] 1. General: Several of the words listed are already included in IBM Style. Examples include "agnostic", "client side", "cloud", "colocate", "data center", "DevOps", and so on. It might be a good idea to compare the list with the latest version of the "Word usage" topic (https://www.ibm.com/docs/en/ibm-style?topic=word-usage), and update the IBM Style entries as necessary? **[Volunteer needed]**

- [x] 2. General: Search for occurrences of "UNIX" to ensure that it is always written in all uppercase letters (see the entry for "UNIX" in the "Word usage" topic https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u). **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/163

- [x] 3. General: Search for occurrences of "may" that should be changed to "can" or "might" -- see the entry for "may" in the "Word usage" topic (https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m). **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [x] 4. General: Search for occurrences of "acronym" to ensure that they are all actual acronyms according to the IBM Style definition (https://www.ibm.com/docs/en/ibm-style?topic=grammar-abbreviations) -- many are initialisms, which are not pronounced as words. **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 5. General: Be consistent in the descriptions of related words:
    (a) Example 1: KB, MB, GB (see https://www.ibm.com/docs/en/ibm-style?topic=measurement-units#si-multiplier-prefixes and https://www.ibm.com/docs/en/ibm-style?topic=measurement-units#multiplier-prefixes-for-bits-and-bytes).
    (b) Example 2: online backup, offline backup.
    In any case, these entries might not be needed -- IBM Style already has entries for "online" and "offline" (see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o). 
  **[Volunteer needed]**

- [x] 6. https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences
Typo: "prerequisties". **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 7.  https://redhat-documentation.github.io/supplementary-style-guide/#release-notes
    (a) Maybe rephrase the Enhancement example to remove the 'Use with caution' term "resides" (see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r)?
    (b) Maybe rephrase the "Known issue" example to remove "There is"? (see https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences)
  **[Volunteer needed]**

- [ ] 8.  https://redhat-documentation.github.io/supplementary-style-guide/#links
    (a) Rephrase the second bullet item, to put "When not using running text," at the start of the sentence?
    (b) In "Example: Running text", second sentence: The pronoun "this" refers to an unstated noun (see https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references, fourth example). **[Volunteer needed]**

- [x] 9.  https://redhat-documentation.github.io/supplementary-style-guide/#_b
In the "backtrace" entry, the comma after traceback should be moved after the quotation marks ("traceback", instead of "traceback,"). **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 10.  https://redhat-documentation.github.io/supplementary-style-guide/#_b
I think the "because" entry should be changed in one of the following ways:
  - Rename to "since" and move to the "s" section. I think it's the term "since" that is generally misused, not the term "because"?
  - Remove entirely, because (sic!) a "since" entry already exists in the "Word usage" topic in IBM Style.
(see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s) 
  **[Volunteer needed]**

- [x] 11.  https://redhat-documentation.github.io/supplementary-style-guide/#_d
Typo in the "DVD writer" entry: "adevice" should be "a device". **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 12.  https://redhat-documentation.github.io/supplementary-style-guide/#_i
ISeries (noun): The text in the iSeries entry is confusing. The incorrect term ("iSeries") is used, but it is given the green icon, and "Use it" is set to "Yes".
A few corrective actions are required (see the pSeries and zSeries entries for comparison):
    (a) The name for the "good" entry should be changed to the approved name "IBM eServer System i".
    (b) A "Do not use" entry should be added for "iSeries" (note that the first letter should be lowercase). 
  **[Volunteer needed]**

- [x] 13.  https://redhat-documentation.github.io/supplementary-style-guide/#_i
IT, I.T. (noun): For clarity, insert a comma after "are used". Otherwise, the reader might misread the text as "all uppercase letters are used to clarify" instead of "Use ... with periods ... to clarify". **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [x] 14.  https://redhat-documentation.github.io/supplementary-style-guide/#_k
KB (noun): Typo: "acromym" (in any case, it's an initialism, not an acronym). **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 15.  https://redhat-documentation.github.io/supplementary-style-guide/#_m
media (noun): Currently includes "diskettes" as an example, but other entries suggest that "diskette" should not be used -- we should be consistent?
If necessary, perhaps suggest a change to the existing IBM Style advice re. "floppy disk" and "diskette"? **[Volunteer needed]**

- [ ] 16.  https://redhat-documentation.github.io/supplementary-style-guide/#_n
numbers (adverb): Some of the examples are labelled with "for example", and some are not -- should be consistent? **[Volunteer needed]**

- [ ] 17.  https://redhat-documentation.github.io/supplementary-style-guide/#_p
PC (noun): This entry refers to IBM Style for further information, but I'm not sure where in IBM Style such information is provided? **[Volunteer needed]**

- [ ] 18.  https://redhat-documentation.github.io/supplementary-style-guide/#_p
pop-up (noun): See the advice in IBM Style (https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#menu). **[Volunteer needed]**

- [ ] 19.  https://redhat-documentation.github.io/supplementary-style-guide/#_r
return (verb): Use initial capitals for key names (see https://www.ibm.com/docs/en/ibm-style?topic=elements-keyboard-keys#key-names). **[Volunteer needed]**

- [x] 20.  https://redhat-documentation.github.io/supplementary-style-guide/#_r
rollout (noun): The "See also" section refers to an entry that doesn't exist -- the "roll-out" entry links back to the original ("rollout") entry. **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 21.  https://redhat-documentation.github.io/supplementary-style-guide/#_s
segmentation fault (noun): In the last sentence, incorrect placement of "Only" (see https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only)? Perhaps rephrase as follows: 'Do not use the abbreviation "segfault" unless absolutely necessary, ...' **[Volunteer needed]**

- [ ] 22.  https://redhat-documentation.github.io/supplementary-style-guide/#_s
Shadowman (noun): In the "See also" section, maybe link directly to the page that refers to Shadowman (https://www.redhat.com/en/about/brand/standards/history) instead of to the top-level brand page (https://www.redhat.com/en/about/brand/standards)? **[Volunteer needed]**

- [ ] 23.  https://redhat-documentation.github.io/supplementary-style-guide/#_s
spec file (noun): Should the references to "RPM" be changed to "RPM package" (see https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories, sixth bullet point)? **[Volunteer needed]**

- [ ] 24.  https://redhat-documentation.github.io/supplementary-style-guide/#_s
straightforward (adjective): Should the usage advice be changed to "Use with caution", similar to the entry for "easy" in the "Word usage" topic (see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e)? **[Volunteer needed]**

- [x] 25.  https://redhat-documentation.github.io/supplementary-style-guide/#_t
thin-provisioned (adjective): The "Incorrect forms" section includes the correct form ("thin-provisioned") as the first entry. **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [x] 26.  https://redhat-documentation.github.io/supplementary-style-guide/#_u
uninterruptible (adjective): Typo: "uninterruptable". **[Breda McColgan]**
https://github.com/redhat-documentation/supplementary-style-guide/pull/166

- [ ] 27.  https://redhat-documentation.github.io/supplementary-style-guide/#_w
we suggest (verb): We might want to remove the references to "recommend" (see https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations, seventh bullet point)? **[Volunteer needed]**

### PR DESCRIPTION
**Pull request by CBID2**
## Description 
This PR removes unnecessary information and incorrect forms for terms such as "agnostic", "client-side", "cloud", "colocate", "data center", and "DevOps". The changes ensure consistency and accuracy in the usage of these terms.

## Issue
Addresses item 1 for #147 

## Note to reviewers
Hi @bergerhoffer! :) I could not access the IBM link mentioned in the first checkbox in issue #147, so I'm not sure if other words need to be removed besides the ones you listed.

### CODE REVIEW

**Review by bergerhoffer — APPROVED** [review]
This looks great, thanks @CBID2!

@redhat-documentation/ccs-style-council Need a few more reviews please!

**Comment by CBID2** [comment]
> This looks great, thanks @CBID2!
> 
> 
> 
> @redhat-documentation/ccs-style-council Need a few more reviews please!

Thanks @bergerhoffer! I made another PR too, so can you check that one as well please?

**Review by joaedwar — APPROVED** [review]
lgtm

**Review by dfitzmau — APPROVED** [review]
I checked a generated preview of changes in the commit and all target instances are removed from the doc.
